src/coreclr/jit/emitxarch.cpp

```diff
@@ -384,7 +384,7 @@ bool emitter::DoesWriteSignFlag(instruction ins)
 bool emitter::DoesResetOverflowAndCarryFlags(instruction ins)
 {
     insFlags flags = CodeGenInterface::instInfo[ins];
-    return (flags & (Resets_OF | Resets_CF)) == (Resets_OF | Resets_CF);
+    return (flags & (Resets_OF | Resets_CF | Writes_SF)) == (Resets_OF | Resets_CF | Writes_SF);
```

Comment:
Should this change be undone? Seems like the other one subsumes it?

Reply:
Removed the redundant code.